### DISCUSSION TITLE
Please how can i deploy on azure

### DISCUSSION THREAD

**Discussion by salahhusa9** [devlikeapro/waha#279] [Ideas]
I want to deploy on azure, but i have problem

**Comment by allburov** [comment]
Which one?

**Reply by salahhusa9** [reply]
Web App DOCKER, i face problem of port, but i solved bay change port.
what is the best recourses for run this project ?

**Reply by allburov** [reply]
Any VPS works, like digitalocean or even laptop always online for small chatbot :)

**Reply by salahhusa9** [reply]
yah, just like the best configs X in RAM  and X CPU ...

**Reply by allburov** [reply]
Oh, I see. It depends how many sessions you'll run,what engine you'll use and the messages you'll receive and send

Here's some advices
https://waha.devlike.pro/docs/overview/faq/#how-much-cpu-and-memory-does-whatsapp-http-api-consume

We'd recommend to have at least 2GB RAM (or 4 is even better) and 2CPU for a single session and start increasing by 0.2-0.5RAM and 0.2-0.5 CPU for every additional session

**Reply by salahhusa9** [reply]
hhhhhhhhhhh `Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz` my `Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz   2.30 GHz` it's the same of my laptop.
me i use WEBJS as engine, if you want to do for you benchmark on cloud (azure) i don't have problem, how can i know the resource RAM/CPU is good?
